lib/matplotlib/tests/test_mathtext.py

```diff
@@ -347,7 +348,8 @@ def test_fontinfo():
         'unknown symbol',
         'double superscript',
         'double subscript',
-        'super on sub without braces'
+        'super on sub without braces',
+        'unclosed group'
```

Comment:
So next change doesn't need to modify two lines like this time:
```suggestion
        'unclosed group',
```